blocks/bricks/simple.py

```diff
@@ -284,6 +284,29 @@ def apply(self, input_):
         return tensor.switch(input_ > 0, input_, 0)
 
 
+class LeakyRectifier(Activation):
+    r"""Like Rectifier, but instead does:
```

Comment:
Please take a look at [numpy docstring format](https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt).

It should one line short description, than a long description.